maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
+- `c` tags in 1.21+ NeoForge should be kept in sync with Fabric's `c` tags as close as possible. New `c` tags or changes to existing `c` tag should by ran by Fabric to ensure they will agree to make the same changes. `c` tags are to be a standard between loaders with Fabric being our main partner in this effort.
```

Comment:
This is not something important enough to have in this document IMO.

Reply:
If we don't write it down here, where do we document that we have to keep c tags in sync with fabric? It's currently and unspoken rule and if you and I were to get hit by a bus, how do we know the next maintainer wont start changing c tags without checking with fabric?